source/app/controllers/urls_controller.rb

```diff
@@ -1,2 +1,26 @@
 class UrlsController < ApplicationController
+  include UrlsHelper
+
+  def index
+  	@urls = Url.all
+  	@url = Url.new(address: "https://")
+  end
+
+  def create
+  	@url = Url.new(url_params)
+  	@url.click_count = 0
```

Comment:
If you put a default value in the database, ActiveRecord will pick it up and you won't need this line.

Reply:
Good call!